src/bundle/Resources/public/scss/core/_custom.dropdown.scss

```diff
@@ -3,7 +3,7 @@
         position: relative;
         color: $ez-color-base-dark;
         font-weight: 700;
-        cursor: pointer;
+        cursor: initial;
```

Comment:
Do we need `initial`? Cannot we just remove it?

Reply:
If you don't need it, then remove it.

Reply:
It overrides: `.ez-field-edit--ezselection .ez-data-source` which has `cursor: pointer;`.

Reply:
So, maybe you should fix it there, hm?

Reply:
I didn't wanted to break something.

Reply:
`ez-custom-dropdown` comes from `ezselection` fieldtype. You can remove it safely.